fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetadata.java

```diff
@@ -2,15 +2,34 @@
 
 package com.starrocks.connector.hive;
 
+import com.starrocks.catalog.Table;
 import com.starrocks.common.DdlException;
 import com.starrocks.connector.ConnectorMetadata;
 import com.starrocks.external.hive.HiveMetaCache;
+import com.starrocks.external.hive.HiveTableName;
 import com.starrocks.server.GlobalStateMgr;
 
+import java.util.List;
+
 public class HiveMetadata implements ConnectorMetadata {
-    private HiveMetaCache metaCache;
+    private final HiveMetaCache metaCache;
 
     public HiveMetadata(String resourceName) throws DdlException {
```

Comment:
this class has not been called now am i right? if this metadata do not used by internal HiveTable, how about passing resource name with a fixed string?

Reply:
different resource name is necessary.
In HiveRepository. there are many map has key which using 'resourceName'. like hiveMetaClient and hiveMetaCache.